scripts/ethcommands.ts

```diff
@@ -77,14 +82,42 @@ async function bridgeNativeToken(argv: any, parentChainUrl: string, chainUrl: st
     const sleep = (ms: number) => new Promise(r => setTimeout(r, ms));
     while (true) {
       const balance = await bridger.getBalance()
-      if (balance.gte(ethers.utils.parseEther(argv.amount))) {
+      if (balance.gte(0)) {
```

Comment:
.gte(depositAmount)?

gte(0) will not work for 2nd transfer, gte(depositAmount) will at least work if 2nd transfer is larger than previous balance which seems reasonable.

there are probably better ways

Reply:
In case when fee token has non-18 decimals, depositAmount is not equal to the amount minted on L2 (we do inflation/deflation to match 18-decimals denomination for native currency). I've improved this check to properly wait for expected amount to be minted on L2.